src/app/(app)/api/stripe/webhooks/route.ts

```
@@ -64,13 +68,28 @@ export async function POST(req: Request) {
+                    await payload.update({
+                        collection: "tenants",
+                        where: {
+                            stripeAccountId: {
+                                equals: data.id
+                            }
+                        },
+                        data: {
+                            stripDetailsSubmited: data.details_submitted
+                        }
+                    })
                 default:
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)